app/src/services/api.ts

```diff
@@ -289,6 +291,18 @@ export const api = createApi({
         "YearlyReportResults",
       ],
     }),
+    deleteInventory: builder.mutation<
+      InventoryAttributes,
+      InventoryDeleteQuery
+    >({
+      query: (data) => ({
+        url: `/inventory/${data.inventoryId}`,
+        method: "DELETE",
+      }),
+      transformResponse: (response: { data: InventoryAttributes }) =>
+        response.data,
+      invalidatesTags: ["InventoryProgress", "InventoryValue", "Inventories"],
```

Comment:
Technically this also invalidates Inventory, ActivityValue, InventoryValue, ReportResults, YearlyReportResults...

Probably won't cause issues for now as you will be on another inventory page or onboarding afterwards but still 😅